posts/2024-11-01-jupyter-book-2-alpha.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Announcing Jupyter Book 2 alpha
```

Comment:
Remove any language like `2.0.0a0`. That is engineer language, user language is `Jupyter Book 2 alpha`

Reply:
```suggestion
title: Announcing the Jupyter Book 2 alpha
```